lua_modules/ds3231/ds3231.lua

```diff
@@ -5,6 +5,8 @@
 -- Tobie Booth <tbooth@hindbra.in>
 --------------------------------------------------------------------------------
 
+require("bit")
```

Comment:
> Wouldn't search for `bit.lua` in the path?

Not sure about the precedence, would need to check this.

> wouldn't you have to require `i2c` and any other NodeMCU module

You don't *have* to but require'ing them at the beginning makes it more transparent that these modules are used somewhere in the code.